### PR TITLE
Animals Table

### PR DESCRIPTION
**Pull request by HansZizold**
** CHANGES **
vet_clinic database created.
The animals' table was created with an id attribute as the primary key.
Animals data inserted and queries executed.
All the data is included in the repo according to the instructions.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+  id integer NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name TEXT,
```
Comment (mikethreels) [inline schema.sql:3]: - [ ] the name should be `varchar()`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Hans, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (HansZizold) [reply schema.sql:3]: ok, I understand. Thank you for your kind feedback.

**Comment by HansZizold** [comment]
Hi @mikethreels, the error was corrected.
I included the ALTER command in the schema file. 
Also, I included a new screenshot with the new table.

**Review by ShahierNashaat — APPROVED** [review]
Hi @HansZizold, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**